components/builder-jobsrv/src/config.rs

```diff
@@ -42,6 +42,8 @@ pub struct Config {
     pub log_path: PathBuf,
     /// Max time (in minutes) allowed for a build job
     pub job_timeout: u64,
+    /// What this server should do when it starts up
+    pub action: String,
```

Comment:
I don't think we should (or need to) put the action inside of the config struct.  We should be able to handle migrate sub-command as a special case, instead of plumbing it all the way into the JobSrv application.

Reply:
See my comment below for why this ended up as part of the config struct.